### ISSUE TITLE
waveform preview always become maximize after render

### ISSUE THREAD

**Issue by jimmysitu** [bmpenuelas/waveform-render-vscode#13]
Hi, All

Each time I update the json file and do the **Waveform Render: Draw**, the preview window will become maximize and full of my vscode screen. This is even annonying when the **Toggle Live Preview**, Anyone could help?

I test this on a Linux, VScode 1.63.0

THanks

**Comment by Nabav** [comment]
For me, it is not maximized. Keeps the layout perfectly. Tested Linux and 1.63.2.

**Comment by jimmysitu** [comment]
Hi, All, Add more info about my vscode and OS
```
Version: 1.63.2
Commit: 899d46d82c4c95423fb7e10e68eba52050e30ba3
Date: 2021-12-15T09:39:46.686Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Linux x64 5.4.0-96-generic
```
```
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.3 LTS
Release:	20.04
Codename:	focal
```

**Comment by jimmysitu** [comment]
Hi, All

I found this issue is caused by reveal() when the panel update.
https://github.com/bmpenuelas/waveform-render-vscode/blob/71f2e4a1e22d90d94a7c5f4348a6ddc657ea8190/src/extension.ts#L59
When I comment out this line, everything works fine.
Any idea?

**Comment by jchabloz** [comment]
I have the exact same issue that started with one of the latest updates of VScode. Same version as @jimmysitu but on Windows 10...

**Comment by jimmysitu** [comment]
@jchabloz  would you like to try my fix and check if works on Windows 10?
It can be download here
https://github.com/jimmysitu/waveform-render-vscode/releases/tag/v0.18.1

Thanks

**Comment by bmpenuelas** [comment]
This issue fixed is included in 9026ceb . The new release is already available in the marketplace. Thanks @jimmysitu for the proposed fix!
